tensorflow/lite/python/lite_test.py

```diff
@@ -751,6 +751,20 @@ def testTFLiteGraphDef(self):
                      output_details[3]['name'])
     self.assertTrue(([1] == output_details[3]['shape']).all())
 
+  def testLiteUnSupportedOps(self):
+    # Tests the object detection model that cannot be loaded in TensorFlow.
+    self._initObjectDetectionArgs()
+
+    # Make the custom_ops as false
```

Comment:
nit: Change comment to:
"`custom_ops` is not set to `True`"

Reply:
Thanks, i have updated the code as per comments